Gems/MachineLearning/Code/Platform/Android/PAL_android.cmake

```diff
@@ -0,0 +1,4 @@
+
+set(PAL_TRAIT_MACHINELEARNING_SUPPORTED TRUE)
+set(PAL_TRAIT_MACHINELEARNING_TEST_SUPPORTED TRUE)
+set(PAL_TRAIT_MACHINELEARNING_EDITOR_TEST_SUPPORTED FALSE)
```

Comment:
Maybe I'm missing something, but it looks like the traits are set the same on all platforms? If so, why have platform-specific build files at all?

Reply:
Comes from the GEM template.. it's all created by default for some reason.

Reply:
They are ament to changed by the Gem author to allow turning on/off GoogleTest support on per platform basis.
For example, to turn on test support for Editor Tests support Windows, then updating the trait in PAL_windows.cmake can be done.